app/jobs/appointment_reminders/send_daily_reminders_job.rb

```diff
@@ -0,0 +1,15 @@
+class AppointmentReminders::SendDailyRemindersJob < ApplicationJob
+  queue_as :high
+
+  def perform
+    # add feature flag?
```

Comment:
While there shouldn't be any reminders to send in other countries, I think it probably makes sense to either feature flag this or guard based on country.